h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -55,6 +58,22 @@
           new StepDefinition(Algo.StackedEnsemble.name(), Alias.defaults),
   };
 
+  private static LeaderboardExtensionsProvider createLeaderboardExtensionProvider(AutoML automl) {
```

Comment:
@seb-h2o is it a sort of default extension provider? if yes maybe it is better to reflect it in the naming?

Reply:
it's not a default in the sense that it would be possible to change it... it is the provider used by AutoML. dot.